src/Ledger/Ledger/Properties.agda

```diff
@@ -134,6 +134,12 @@ module _ where
 
 -- ** Proof that the set equality `govDepsMatch` (below) is a LEDGER invariant.
 
+-- Mapping a list of `GovActionID × GovActionState`s to a list of
```

Comment:
It's very interesting to me that you're proving things about a more complicated relation now, but the proof has gotten a lot shorter.

Reply:
New gov actions used to be appended to the end of the GovState.  Now we insert according to the priority.  So I don't think it's much more complicated, though all the lemmas about list concatenation became pretty useless.  Also, this change gave me another chance to analyze, rework, and simplify the proof.